examples/code/gc/barrier_bench/barrier_bench.sh

```diff
@@ -0,0 +1,110 @@
+### run
+  $ jbuilder build barrier_bench.exe
+%% --non-deterministic [skip]
+  $ ./_build/default/barrier_bench.exe -ascii alloc -quota 1
+  Estimated testing time 2s (2 benchmarks x 1s). Change using -quota SECS.
+
+    Name        Time/Run   mWd/Run   mjWd/Run   Prom/Run   Percentage
+   ----------- ---------- --------- ---------- ---------- ------------
+    mutable       8.98ms    2.00Mw     20.36w     20.36w      100.00%
+    immutable     5.66ms    5.00Mw                             63.08%
+
+### help
```

Comment:
The help generation here seems awfully big. Probably an ellipsis is desired, rather than printing out the entire help text, no?

Reply:
As far as I know, there is no first-class support for ellipsis. But something like adding `...` and non-deterministic annotations would work -- is it what you had in mind?